blog/2015-02-26-Getting-Started-With_Docker-And-Wildfly.adoc

```diff
@@ -0,0 +1,195 @@
+changes the container makes to the folder can be visible on the host, and vice versa. 
+
+*Mapping Folders may cause IO errors for SELinux!*  To ensure your container can actually read and write to the folder, 
+you'll need to run `setenforce 0` to disable SELinux, or, alternatively, give Docker the permissions and exceptions in SELinux. 
+
+....
+[root@localhost jbds_wf_external]$  mkdir /home/rob/tmp/dockertest1
+[root@localhost jbds_wf_external]$  docker run -it -p 8080:8080 -p 9990:9990 \
+-v /home/rob/tmp/dockertest1:/opt/jboss/wildfly/standalone/deployments/:rw  jboss/wildfly
+....
+
+If you were to now place a .war file inside `/home/rob/tmp/dockertest1`, it will be picked up by the deployment scanner, 
+and visible in a web browser. 
+
+=== Making your Server Adapter
+
+The final step of this example is to create your Wildfly 8.2 server adapter in JBossTools, 
+and to create and deploy a web application to your temporary folder, which in my case is
+`/home/rob/tmp/dockertest1`
+
+First, we'll open the `Servers View` and create a new `Wildfly 8.x` server adapter.
+Since we're exposing our container's ports on our localhost, we can leave the host as 
+`localhost`. Make sure your server is marked as `Local` and Controlled by `Filesystem and shell operations` 
+as shown below. 
+
+image::images/20150226_docker/local_fs_1.png[]
+
+Since we're hacking a Local Filesystem server adapter to work for Docker, we'll still need a local
+runtime in this example, so there's no real way to avoid that. 
+
+=== Configuring your Server Adapter
+
+Once your server is created, you'll find it in the `Servers View`, where we can double-click
+it to open the `Server Editor`.  From here, we can make what configuration changes we'll need. 
+First, we'll need to make sure the server is `Externally Managed`.  This means 
+our attempts to start and stop it in Eclipse will be mostly ignored, since we're
+expecting the user to handle that via Docker. 
+
+image::images/20150226_docker/local_fs_externally_managed.png[]
+
+Next, we'll need to disable the tooling for keeping deployment scanners in sync with
+the locations JBossTools expects to be deploying. Since we've already mapped the folder
+in via the Docker command line, we won't need any additions to the deployment scanners at all. 
+
+image::images/20150226_docker/local_fs_scanners.png[]
+
+And finally, on the `Deployment` tab of the Server Editor, we'll want to 
+mark the default deploy folder to be a *Custom* location, and choose the folder
+that we previously mapped in via Docker's command line, as shown below:
+
+image::images/20150226_docker/local_fs_deploy.png[]
+
+
+Once all this is done, we can save the editor, and our server adapter is configured properly. 
+
+=== Make a Web Project
+
+In this example, we can create a very simple web project by browsing to
+`File -> New -> Dynamic Web Project`, Once the web project is created, we can
```

Comment:
could also suggest using the HTML 5 app from JBoss Central which actually would provide some default content.